cross/systemd/Makefile

```diff
@@ -0,0 +1,59 @@
+PKG_NAME = systemd
```

Comment:
I guess you do not expect to provide "systemd" on its whole on DSM 6 (or 7) but only grab "udev" support as far as I understand. If so, I propose to name it as `cross/systemd-udev` or other relevant postfix to identify concerned subsystem.

Reply:
agreed, make sense.